GOVERNANCE.md

```diff
@@ -0,0 +1,91 @@
+
+## Technical Steering Committee
+
+A subset of the collaborators forms the Technical Steering Committee (TSC). The TSC has authority over the following aspects of this project:
+
+- Technical direction and guidance
+- Project governance and process 
+- Contribution policy
+- Conduct guidelines
+- Maintaining the list of collaborators
+
+The current list of TSC members is maintained TBD
+
+### Scope
+
+The TSC is primarily responsible for the Aerie core and UI projects:
+
+- https://github.com/NASA-AMMOS/aerie
+- https://github.com/NASA-AMMOS/aerie-ui
+
+However, the TSC also has responsibility over some projects, which are dependencies of the core projects:
+
+- https://github.com/NASA-AMMOS/aerie-docs
+- https://github.com/NASA-AMMOS/aerie-gateway
+- https://github.com/NASA-AMMOS/aerie-ts-user-code-runner
+- https://github.com/NASA-AMMOS/aerie-ampcs
+
+Addtionally, the TSC has authority over a few other "Aerie Extended Universe" projects:
+
+- https://github.com/NASA-AMMOS/aerie-cli
+- https://github.com/NASA-AMMOS/aerie-mission-model-template
+- https://github.com/NASA-AMMOS/aerie-lander
+
+### Decision Making Process
+
+Any community member can create a GitHub issue or comment asking the TSC to review something. Prior to implementing a substantial contribution, the design of that contribution should be reviewed by at least one member of the TSC. If consensus-seeking fails during the review of a pull request or in design discussions, the issue will be addressed by the TSC to make a determination on direction. TSC members will meet regularly and will keep track of decisions made when consensus was not met. 
+
+The TSC can nominate new members at any time. Candidates for membership tend to be collaboraters who have shown great dedication to the project and/or experts in a particular domain or project component. TSC members are expected to be active contributors or members who have made significant contributions within the past 12 months. 
+
+## Project Management Committee 
+
+The Project Management Committee (PMC) is made up of sponsor organization representives (i.e. those providing funding to the project) and key stakeholders who rely or plan to rely on Aerie to meet a crticial need (e.g. project using Aerie for space mission operations). The PMC has the following primary responsibilities
+
+- Maintaining the overall project roadmap
+- Determining project requirements and committments to sponsors and stakeholders
+- Assessing available resources and allocating them across the project
+- Monitoring and reporting on progress against the roadmap 
+- On-boarding new sponsors and stakeholders
+- Addressing any legal considerations
+
+The current list of PMC members is maintained TBD. If your project or organization is planning to use Aerie and you would like to join the PMC, please contact TBD....  

```

Comment:
Two TBDs in this line.

Reply:
I went ahead and removed the TBDs although the exact PMC members are still in work. I am also debating where to put the PMC list. I am thinking either inline here or in the README. Any preferences?